conans/test/unittests/client/rest/uploader_test.py

```diff
@@ -15,12 +15,14 @@ class UploaderUnitTest(unittest.TestCase):
     def test_401_raises_unauthoirzed_exception(self):
 
         class MockRequester(object):
+            retry = 0
```

Comment:
Is this needed in the current implementation? Are the retry finally members of the requester?